canonical-web-and-design/vanilla-framework
# Use run script v3.4.1

QA
--

Check commands work:

``` bash
./run clean
./run build
./run yarn build-docs
./run
```

Browse the docs at http://127.0.0.1:8101.

``` bash
docker build --build-arg REVISION_ID=nope --tag docs docs
docker run -ti -p 8301:80 docs
```

Browse the docs at http://127.0.0.1:8301.
